db/query.py

```diff
@@ -438,3 +438,9 @@ def get_all_apps_with_groups() -> List[AppToGroup]:
     """
     session = Session()
     return session.query(App, Group).filter(Group.id == AppToGroup.group_id).filter(App.id == AppToGroup.app_id).all()
+
+def remove_employee_role_by_id(employee_id: str) -> bool:
```

Comment:
This method is not necessary since we can use the roles attribute in an employee object to remove an employee's relationship to a role.